src/lib/flex/flex/flex.ts

```diff
@@ -274,8 +274,11 @@ export class FlexDirective extends BaseDirective2 {
 
   /** Trigger a style reflow, usually based on a shrink/grow input event */
   protected triggerReflow() {
-    const parts = validateBasis(this.activatedValue, this.flexGrow, this.flexShrink);
-    this.marshal.updateElement(this.nativeElement, this.DIRECTIVE_KEY, parts.join(' '));
+    const activatedValue = this.activatedValue;
+    if (activatedValue) {
+      const parts = validateBasis(activatedValue, this.flexGrow, this.flexShrink);
```

Comment:
Note to self, this should be a check for undefined, because the empty string is valid here